pkgs/development/python-modules/crytic-compile/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonPackage, fetchPypi, python3Packages }:
+
+buildPythonPackage rec {
+  pname = "crytic-compile";
+  version = "0.1.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0s4nr3jrm7665qyd04j3rxij0a58dwrfvc0sn05c7c5zwr0x155s";
+  };
+
+  buildInputs = with python3Packages; [ pysha3 ];
```

python packages are added to propagatedBuildInputs
```suggestion
  propagatedBuildInputs = [ pysha3 ];
```

---

Done